substrate/frame/paged-list/Cargo.toml

```
@@ -18,13 +18,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 codec = { features = ["derive"], workspace = true }
 docify = { workspace = true }
 scale-info = { features = ["derive"], workspace = true }
-
-frame-benchmarking = { optional = true, workspace = true }
-frame-support = { workspace = true }
-frame-system = { workspace = true }
-
-sp-core = { workspace = true }
-sp-io = { workspace = true }
+frame = { workspace = true, features = ["experimental", "runtime"] }
```

frame no longer require experimental feature, though you might need to merge master branch
```suggestion
frame = { workspace = true, features = ["runtime"] }
```